### ISSUE TITLE
Pane separation algorithm to be modified to use TSV

### ISSUE THREAD

**Issue by AnshDubey1999** [UAlbertaALTLab/mobidict-android#51]
Currently, the pane separation algorithm uses the paradigm-panes output to separate panes. Change this to use TSV.

**Comment by AnshDubey1999** [comment]
An explanation by Antti:

An additional note (that I mentioned earlier): The heading tab for a pane should be based on whatever column labels are in the topmost cell, per each pane. E.g. for the VAI.tsv pane specs, for the Core pane the heading is | Core, and then for the next panes they are | Prs | Ind, | Prs | Cnj, | Prt | Ind, | Prt | Cnj, | Fut+Int, and so forth.

But even in the single-column layout, a pane can sometimes have internal column labels. This concerns mainly the Core panes. For the VAI.tsv pane specs, you'll see that it internally has three column labels, | Prs+Ind, | Prs+Cnj, and | Imp. This is because the selected individual forms for the Core pane can come from multiple other panes.

Specifications of panes are separated by one or more empty lines.

The topmost label, which can consist of multiple parts, e.g. | X, or | X | Y, or | X | Y | Z, is what should be interpreted as the pane headings. If there are multiple components, they should get relabelled with the longest match principle, i.e. if there is a relabeling for X+Y+Z, then that should be used, but if not, one should look for a relabeling of X+Y or Y+Z, and if not, then relabel the feature tags X, Y, and Z individually.

After the heading label(s), the pane consists of the content below, until an empty line. And under the heading label (the first cell), the pane may contain internal column labels, marked with | X or | X | Y.

I suppose we could mark the pane headings with a separate indicator, such as #. However, in the general specification of panes, we did consider the possibility that individual panes could sometimes be aggregated, when a device has a broad screen. In such a case, the non-shared elements of the pane heading could instead be interpreted as a column label for each individual pane, within the multi-pane layout. While I believe this is supported for the on-line version of the panes that the students wrote, as we've discussed earlier I would not worry about the possible implementation of the panes with a single-column-pane layout for mobile devices (that largely have narrow screens).
